src/content/release/breaking-changes/system_context_menu_controller_show.md

```diff
@@ -0,0 +1,117 @@
+`SystemContextMenu.getDefaultItems` to get the default items based on the given
+`EditableTextState`. For example, if the `EditableTextState` indicates that
+there is nothing selected, then the "Copy" button will not be included, since it
+requires a selection to copy.
+
+## Migration guide
+
+Most users use the system context menu through the `SystemContextMenu` widget,
+and in this case there will be no change required. The `SystemContextMenu`
+widget automatically gets the default items under the hood.
+
+No migration is needed:
+
+```dart
+class _MyWidgetState extends State<MyWidget> {
+  @override
+  Widget build(BuildContext context) {
+    TextField(
+      contextMenuBuilder: (BuildContext context, EditableTextState editableTextState) {
+        return SystemContextMenu.editableText(
+          editableTextState: editableTextState,
+        );
+      }
+    );
+  }
+}
+```
+
+For advanced users that directly work with `SystemContextMenuController`,
+migrate to the new method `SystemContextMenuController.showWithItems`. The
+default can be obtained from `SystemContextMenu.getDefaultItems` as a list of
+`IOSSystemContextMenuItem`s, which can be converted to the format required by
+`showWithItems` through `IOSSystemContextMenuItem.getData`.
+
+Code before migration:
+
+```dart
+_controller.show(selectionRect);
+```
+
+Code after migration:
+
+```dart
+final List<IOSSystemContextMenuItem> defaultItems =
+    SystemContextMenu.getDefaultItems(editableTextState);
+final WidgetsLocalizations localizations =
+    WidgetsLocalizations.of(context);
+final List<IOSSystemContextMenuItemData> defaultItemDatas =
+    defaultItems
+        .map((IOSSystemContextMenuItem item) =>
+            item.getData(localizations))
+        .toList();
+_controller.showWithItems(selectionRect, defaultItemDatas);
+```
+
+## Timeline
+
+Landed in version: 3.29.0-0.3.pre<br>
+In stable release: not yet
+
+## References
+
+// TODO(justinmc): Fill this out.
+API documentation:
+
+* [`DropdownMenu`][]
+* [`FocusNode.canRequestFocus`][]
+* [`TextField.canRequestFocus`][]
+* [`TextField.focusNode`][]
+
+Relevant issues:
+
+* [Broken selection on TextField if canRequestFocus: false][]
+* [DropdownMenu Disable text input][]
+
+Relevant PRs:
+
+* [Add requestFocusOnTap to DropdownMenu][]
+* [Replace TextField.canRequestFocus with TextField.focusNode.canRequestFocus][]
+
+[`DropdownMenu`]: {{site.api}}/flutter/material/DropdownMenu-class.html
+[`FocusNode.canRequestFocus`]: {{site.api}}/flutter/widgets/FocusNode/canRequestFocus.html
+[`TextField.canRequestFocus`]: {{site.api}}/flutter/material/TextField/canRequestFocus.html
+[`TextField.focusNode`]: {{site.api}}/flutter/material/TextField/focusNode.html
+
+[Native ios context menu]: {{site.repo.flutter}}/pull/143002
+[Secure paste milestone 2]: {{site.repo.flutter}}/pull/159013
+[[ios][secure_paste]show menu item based on info sent from framework]: {{sites.repo.engine}}/pull/161103
```

Comment:
Does this item belong in the bulleted list above? It's malformed, at any rate, and is breaking the build.

Reply:
Should be fixed now. I had copied some of this stuff from another migration guide and left this as a TODO.